testsuite/forge-cli/src/main.rs

```diff
@@ -1532,10 +1532,10 @@ fn realistic_env_max_load_test(
                 .add_latency_threshold(3.4, LatencyType::P50)
                 .add_latency_threshold(4.5, LatencyType::P90)
                 .add_latency_breakdown_threshold(LatencyBreakdownThreshold::new_strict(vec![
-                    (LatencyBreakdownSlice::QsBatchToPos, 0.3),
-                    (LatencyBreakdownSlice::QsPosToProposal, 0.25),
+                    (LatencyBreakdownSlice::QsBatchToPos, 0.35),
```

Comment:
I've seen in a couple cases where the value gets pretty high, but drops back down. (Not sure why, might need some digging into.) In these case, I think we need to use a max_breach_pct to avoid failures (10% seems like it would work.)

Examples (have to expand the comment):
* https://github.com/aptos-labs/aptos-core/pull/9482#issuecomment-1665798925
* https://github.com/aptos-labs/aptos-core/pull/9497#issuecomment-1666061643

Reply:
yeah, I want to have stats that I print here, and then I'll probably have a limit for both max and p75/p90.
but don't want to make more noise without having better data.

Reply:
Sounds reasonable!